terraform/aws/environments/test/4-command-and-control/3-cluster-post-installation/main.tf

```diff
@@ -5,6 +5,9 @@ terraform {
     key    = "mattermost-central-command-control-post-installation"
     region = "us-east-1"
   }
+  required_providers {
+    helm = "~> 0.10"
+  }
```

Comment:
@angeloskyratzakos can we add the pin down for staging and prod as well?

Reply:
Done I have pinned down prod and staging as well.